cis194/week7/zehua/stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```

Comment:
do we usually check in the lock file?

Reply:
https://github.com/commercialhaskell/stack/issues/4795